docs/theme-context.rst

```diff
@@ -0,0 +1,117 @@
+Read the Docs data passed to Sphinx build context
+=================================================
+
+Before calling `sphinx-build` to render your docs, Read the Docs injects some
+extra context in the templates by using the ``html_context`` Sphinx setting in
+the ``conf.py`` file. This extra context allows Read the Docs to add some
+features like "Edit on GitHub" and use a custom Analytics code, among others.
+
+
+Context injected by default
+---------------------------
+
+Here is the full list of values injected by Read the Docs as a Python
+dictionary. Note that this dictionary is injected under the main key
+`readthedocs`:
+
+
+.. code:: python
+
+    'readthedocs': {
+        'version': {
+            'name': str,
```

Comment:
So on the version data, any reason not to just use a Version object serialized through DRF?